.travis.yml

```diff
@@ -4,6 +4,11 @@ jdk: oraclejdk8
 
 env:
   global:
+    - SONAR_HOST_URL="https://sonarcloud.io"
+
+    # SONAR_LOGIN value
+    - secure: "N/wdh1L9n2kp+BfM8RfmOQQIlbngTKlO5EmnlMh7Q8RXK3N7U/k+74r7yRCqWNJd4zpzKpyQYKYxX6ZkwH8bcJ/SOipDgC9p4yrjY+f+juDK0LrWHg1pqssJL5Q7NTMlaXl5E8q54mCn59t//JrNC2+/KrjoYbJLaRyhaEe0nolAJsxhP1oEjs19Y+QRk9jbpByQLesbVy5LKekxaXQ94wfuPSJPcLDBYh9yQ9aMorqW9Pn2ZZpm7e0AlYjstUoShZY8nb8gsNNpvxrVme9q/1k3POxjbhnauwolc9WNvsFl8UdutCyZsmQbDNEZyJ/R9c9rtrJ5tT/bCZ34zpUX3wH9MjAb2wt+DFHZ7hZT8O5O5lQ1GG6fUZZBm4mT0JNg9HLPRk/TlVDJlmNm4KgzayTTqkD+ugVHNg7CBES5s8Ww9EM+7P2T74/8ZH0UwSBiEWM/cc3vYEpEThnPD5HcUMFGRFJJFAf1UCQAypc5mWFTlCIXuno34a8KHwAS5fyxhHGrIcLQZd1Sq+RIWnKQNekALZMRHQcDB53PGBk4GOtWmf4jAyWBxby/O52g4sJVtoX5jeVnVVPXwSXF2hJFnWDlvLAbV8nNU/PvqjXxS3h11/o11CPwu/vOeHzM5OWTqyCYttKQramr6/qALfW9DIiCr5D+mNu1CETvWFO0FGQ="
```

Comment:
@kzaikin Could you have a try with following parameter? I've hidden `SONAR_LOGIN=token` in this text:

>"FjzPYgjdN9WRvAwvJDFP0MqWMOsqCG0tNO6ts7X1p6HWikpoHnTqWgMl0YOgGK93trvB6vtIisZVVdLZG1rCwnXlO4Xd+6f+nhsJDledd9b4EePQPECJbHZZ8Hg69ATIlm7chll73GZzWTAL2Q3pIdsEJbwtqPzJ/j0H4COkayDeQ4ERA50pvvkXTiQrwxc5URrNH36ZOn10hfUEKO03EKbAlD+5dOzcxqu5WiS7j10zbMBCrKqhzN+zJQSOuDCTQz2S8McJBTiz4kMEBkF1q4Ax+1ywz2vICL3/F5/xqnHvHOk9wZE/4OOiNxjqxVd+Ik6LGE1BbLuATQz0Cmc3DiqsGYBsKtlk88v1QX2hsND/K0hwda/uItzbh95HmnrRYHYhzOcgcIVRqVwoPkYYWPYr9S2f3DGlFEnlGwJHgvqjuwMJaBmm2zblxPammEx3XoWqrKQ7RrelH1tUTWXy/tRss4Fof+D33/NJqa71Xvph1meKMcu+OOVatCul1IJaTtf2pyyifLTNRntgl4C4ok57PN882lRp55/Ifmi+xbt3Nd7lM1Fpy2eDugK1UINmkjRQsyCZY3EVgcM/NkrMa6qHhZFu5F1iv3EBYRiV/WKJ9PtTErkTeX1JZtyZYo7yS+VlIXIJFLoxfrV9TH5B3kczVILxgfJJ6gVPQOY5984="